docs/administration/backup-restore/_index.md

```diff
@@ -0,0 +1,88 @@
+  1. In order to exclude the volume of Redis in backup, we need to label the Redis pod, PVC and PV with specific label:
+     ```shell
+      # label the Pod of Redis, replace the namespace and Pod name with yours
+      kubectl -n harbor label pod/harbor-redis-0 velero.io/exclude-from-backup=true 
+      # label the PVC of Redis, replace the namespace and PVC name with yours
+      kubectl -n harbor label pvc/data-harbor-redis-0 velero.io/exclude-from-backup=true
+      # get the name of Redis PV, replace the namespace and PVC name with yours 
+      kubectl -n harbor get pvc data-harbor-redis-0 --template={{.spec.volumeName}}
+      # label the PV of Redis, replace the pv-name with the one get from last command
+      kubectl label pv/pv-name velero.io/exclude-from-backup=true
+     ```
+  1. Back up Harbor
+      ```shell
+      # replace the namespace and backup name with yours
+      velero backup create harbor-backup --include-namespaces harbor --snapshot-volumes --wait
+      ```
+  
+* Restic
+  1. Exclude the volume of Redis in backup
+      ```shell
+      # replace the namespace and pod name with yours
+      kubectl -n harbor annotate pod/harbor-redis-0 backup.velero.io/backup-volumes-excludes=data
+      ```
+  1. Back up Harbor
+      ```shell
+      # replace the namespace and backup name with yours
+      velero backup create harbor-backup --include-namespaces harbor --default-volumes-to-restic --wait
+      ```
+
+### Unset ReadOnly
+Follow the same steps in [Set Harbor to ReadOnly](#set-harbor-to-readonly), uncheck the **Repository Read Only** check box and click the **Save** button to save the configurations.
+
+## Restore Harbor Instance
+### Restore from the Backup
+```shell
+# replace the backup and restore names with yours
+velero restore create harbor-restore --from-backup harbor-backup --wait
+```
+
+### Unset ReadOnly
+As we set Harbor to ReadOnly when doing the backup, the instance is still in ReadOnly mode after the restoring, follow [Unset ReadOnly](#unset-readonly) to unset ReadOnly.
+
+
+## Limitations
+* **The upload purging process may cause backup failure**  
+  A purging process starts in the `registry` pod by default, it removes the unused files under the upload directory periodically and cannot be disabled without restarting. This may impact the backup when using Restic and cause failures.  
+  It's better to increase the interval of the purging operation(the default value is 168h) and do the backup in the middle of two rounds of purging to avoid files removing.
+* **The data in memory will lose during the backup**  
+  Currently the repository pulls and artifact pull time are put in the memory and synced into the database periodically. So when doing the backup, the data that isn't synced will lose, but this is low impact. 
+* **Tasks may hang in in-progress status**  
+  Tasks(replication/garbage collection/scan/etc.) may hang in in-progress status after restore, stop them on the portal can workaround it.
```

Comment:
how to fix it?

Reply:
The related jobs are gone after restoring, it's just the status hang in in-progress